setup/seed/articles_seed.js

```diff
@@ -124,6 +124,22 @@ export default [
         creationTime: new Date("2024-03-29"),
         modificationTime: new Date("2024-03-29"),
       },
+      {
+        user: new mongoose.Types.ObjectId("b00000000000000000000004"),
+        comment: "Cool article, learned a ton!",
+        commentStatus: CommentStatus.Unresolved.status,
+        creationTime: new Date("2024-03-30"),
+        modificationTime: new Date("2024-03-30"),
+        _id: "66ba07fc440f0ccec39b317d",
```

Comment:
I think in our pattern we opt to use a mongoose ObjectId instead of a string for the `_id` field. Same for the other comment.

Reply:
Weird thing happened. Now, when I run all the tests together the comment status doesn't change, but if I run just the one test to change it then it works. That's the only thing I've changed.